kadok.go

```diff
@@ -109,10 +107,13 @@ func main() {
 	<-sc
 
 	// Cleanly close down the Discord session.
-	dg.Close()
+	err = dg.Close()
+	if err != nil {
```

Comment:
could directly be `if err != dg.Close() {`

Reply:
Following `if` specification this would mean that it checks the previous value  assigned to `err` with the value returned by `dg.Close()` without changing the value of `err`.
The right syntax is `if err = dg.Close(); err != nil {` which does not add much to way it is already written.

For reference, the usages for if are:
- `if  var declaration;  condition { `
- `if  condition { `